epytope/EpitopePrediction/External.py

```diff
@@ -2087,10 +2089,10 @@ def parse_external_result(self, file):
             if not len(row):
                 continue
             
-            if Allele.organism not in row[HLAIndex.NETMHCII_2_2]:
+            if "HLA" not in row[HLAIndex.NETMHCII_2_2]:
```

Comment:
I assume this did not work?

Reply:
Yes unfortunately not..only works using an actual `Allele Obj`